core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -3899,19 +4523,65 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
             if (memory)
                 linear_mem_size = memory->memory_data_size;
 #endif
-            if (wasm_copy_exception(module, NULL))
+            if (wasm_copy_exception(module, NULL)) {
+#if WASM_ENABLE_EXCE_HANDLING != 0
+                /* the caller raised an exception */
+                const char *uncaught_exception = wasm_get_exception(module);
+                _EXCEDEBUG("**** return_func: back from call with %s ****\n",
+                           uncaught_exception);
+
+                // libc_builtin signaled a "exception thrown by stdc++" trap
+                if (uncaught_exception
+                    && strcmp(uncaught_exception,
+                              "Exception: exception thrown by stdc++")
```

Comment:
suggest to use `strstr(uncaught_exception, "exception thrown by stdc++")`

Reply:
Updated according to suggestion.

Reply:
updated accordingly.